lld/MachO/SyntheticSections.cpp

```diff
@@ -1251,26 +1255,26 @@ void SymtabSection::emitStabs() {
 
       // We use 'originalIsec' to get the file id of the symbol since 'isec()'
       // might point to the merged ICF symbol's file
-      symbolsNeedingStabs.emplace_back(defined,
-                                       defined->originalIsec->getFile()->id);
+      Defined *funcBodySym = getFuncBodySym(defined);
+      symbolsNeedingStabs.emplace_back(SymbolStabInfo{
+          defined, funcBodySym->originalIsec->getFile()->id, funcBodySym});
```

Comment:
I could be wrong, but I think this isn't needed.
```suggestion
      symbolsNeedingStabs.emplace_back(
          defined, funcBodySym->originalIsec->getFile()->id, funcBodySym);
```

Reply:
I think that works if `SymbolStabInfo` has a 3-param constructor. As-is it'll give errors like:
```
candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 3 were provided
```